examples/hyperpod-eks/README.md

```diff
@@ -0,0 +1,103 @@
+# Run on AWS SageMaker HyperPod with EKS
+
+This example shows how to run SkyPilot on AWS SageMaker HyperPod with EKS.
+
+## Prerequisites
+
+- An existing SageMaker HyperPod with EKS (or you can create one with AWS [doc](https://catalog.workshops.aws/sagemaker-hyperpod-eks/en-US/00-setup/own-account/01-workshop-infra-script))
+- SkyPilot installed: [installation doc](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)
+```bash
+pip install skypilot-nightly[kubernetes]
+```
+
+
+## Connect to HyperPod cluster
+
+1. Get your EKS cluster name and region from the SageMaker HyperPod console.
+
+```bash
+aws eks update-kubeconfig --name $EKS_CLUSTER_NAME --region $EKS_REGION
+```
+
+2. Connect SkyPilot to the cluster.
+
+```bash
+sky check k8s
+```
+
+![](https://i.imgur.com/aZaocOt.png)
+
+If you are using it with SkyPilot for the first time, you may see a hint to create GPU labels for your nodes. Follow the instructions to create the labels.
+
+```bash
+python -m sky.utils.kubernetes.gpu_labeler --context <your-eks-context>
+```
```

Comment:
```suggestion
```bash
python -m sky.utils.kubernetes.gpu_labeler
```
```

Does `aws eks update-kubeconfig` set the new context as default? If so we may be able to skip the `--context` arg in the command and have it as a optional tip below.

Reply:
Good catch! Yes, it sets the default. Our `sky check`'s hint for the labeller has the context field, we can also consider remove it for simplicity, cc'ing @SeungjinYang. Not urgent : )